ci/jjb/jobs/pulp-backup.yaml

```diff
@@ -0,0 +1,47 @@
+- job-template:
+    name: 'pulp-{pulp_version}-{pulp_build}-backup-{os}'
+    node: '{os}-np'
+    properties:
+        - qe-ownership
+        - build-discarder:
+            days-to-keep: 3
+            num-to-keep: 3
+            artifact-days-to-keep: 3
+            artifact-num-to-keep: 12
+    scm:
+        - pulp-packaging-github
+    wrappers:
+        - config-file-provider:
+            files:
+                - file-id: rhn_credentials
+                  variable: RHN_CREDENTIALS
+        - inject:
+            properties-content: |
+                OS={os}
+                PULP_BUILD={pulp_build}
+                PULP_VERSION={pulp_version}
+    triggers:
+        - timed: '@midnight'
+    builders:
+        - shell:
+            !include-raw-escape:
+                - 'ssh-setup.sh'
+        - shell:
+            !include-raw-escape:
+                - 'pulp-backup.sh'
+        - shell: |
+            if [[ "${{OS}}" =~ "rhel" ]]; then
+                sudo subscription-manager unregister
+                sudo subscription-manager clean
+            fi
+        - capture-logs
+    publishers:
+      - archive:
+          artifacts: '*.bz2'
+      - join-trigger:
+          projects:
+            - 'pulp-2.13-stable-restore-rhel7'
```

Comment:
I was following the example I saw on other jobs of both having the upstream job trigger the downstream job and the downstream job having the reverse trigger. It can be removed if you think that is best.